ttnn/cpp/ttnn/operations/normalization.hpp

```diff
@@ -184,6 +184,9 @@ struct GroupNorm {
         const std::optional<ttnn::DataType> dtype = std::nullopt,
         std::optional<CoreGrid> core_grid = std::nullopt,
         std::optional<bool> inplace = std::nullopt) {
+        if (input_tensor.get_layout() == Layout::TILE) {
+            inplace.value() = false;
```

Comment:
Would you prefer this?
```suggestion
            inplace = false;
```

Reply:
Do you know if/when we are going to support optional output tensor argument and replace `inplace` argument with it?

Reply:
the output tensor is not optional, it is just inplaced with input tensor (using the same L1 address)